.github/workflows/pipeline-indexer-staging-promotion.yml

```
@@ -0,0 +1,15 @@
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  on-merge:
+    if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'pipeline-main-replica'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Echo Hello World
+        run: echo "Hello World"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)